py/mock.py

```diff
@@ -369,7 +369,8 @@ def setup_logging(config_path, config_opts, options):
     try:
         if not os.path.exists(log_ini):
             if os.path.normpath('/etc/mock') != os.path.normpath(config_path):
-                log.warning("Could not find required logging config file: %s. Using default...", log_ini)
+                log.warning("Could not find required logging config file: "\
+                "%s. Using default...", log_ini)
```

Comment:
The 2nd line should be indented, and I don't think the backslash is needed. Also, I think a better place to split would be the comma before log_ini.